crmsh/qdevice.py

```diff
@@ -102,7 +102,6 @@ class QDevice(object):
 
     Whole certification process:
     For init
-    Step 1:  init_db_on_qnetd
```

Comment:
Then the docstring here needs to be adjusted, starts with 'Step 2' looks strange